upload/upload.go

```diff
@@ -189,7 +190,8 @@ func uploadFiles(files, outFiles []string, targetDir string, config *Config) err
 		if err != nil {
 			return err
 		}
-		log.Infof("file uploaded to %s", string(aws.StringValue(&result.Location)))
+		log.Infof("file uploaded to %s\n", string(aws.StringValue(&result.Location)))
```

Comment:
```suggestion
```

Reply:
Some unit tests are using this log to assert the result. I left these with level=Info so they are only available if we raise the loglevel, debugging purposes etc.